content/en/core/overview/roadmap.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Roadmap"
+linkTitle: "Roadmap"
+weight: 1
+description: >
+  Improvements that are being worked on, and those that are being considered next
+aliases:
+  -    /core/overview/technical-roadmap
+relatedContent:
+  core/releases
+---
+
+{{% pageinfo %}}
+The CHT is actively maintained by [Medic](https://medic.org), a non-profit organization that serves as the technical steward, alongside many [partners in the CHT Community]( {{% ref "why-the-cht/#the-cht-community" %}}).
```

Comment:
Do you think this wording is more consistent? It also sounds better to me than maintained which may imply the CHT is not being developed further.

```suggestion
The CHT is stewarded by [Medic](https://medic.org), a non-profit organization alongside many [partners in the CHT Community]( {{% ref "why-the-cht/#the-cht-community" %}}).
```

Reply:
Right, I see what you mean. I find "stewarded" awkward, and not entirely clear, so will make another suggestion. I'll have to make it separately because this thread is marked _outdated_ by accepting the previous suggestion.